vizro-ai/tests/unit/vizro-ai/dashboard/test_pydantic_output.py

```diff
@@ -0,0 +1,28 @@
+import vizro.models as vm
+from vizro_ai.dashboard._pydantic_output import _create_message_content, _create_prompt_template, _get_pydantic_model
+
+
+def test_get_pydantic_model(component_description, fake_llm):
```

Comment:
Can we also test the other flows, when the model fails to create something and it goes into retry mode?